<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-Q8R0VB5" failures="1" tests="2" name="Error Validation Test" time="24.103" errors="0" timestamp="2022-11-16T20:53:34 CST">
  <testcase classname="Ahmad.PersonalFramework.test.ErrorValidation" name="LoginErrorValidation" time="1.245"/>
  <testcase classname="Ahmad.PersonalFramework.test.ErrorValidation" name="ProductErrorValidation" time="1.918">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at AhmadPersonalFramework.PageObjects.ProductCataloguePage.addProductToCart(ProductCataloguePage.java:50)
at Ahmad.PersonalFramework.test.ErrorValidation.ProductErrorValidation(ErrorValidation.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- ProductErrorValidation -->
</testsuite> <!-- Error Validation Test -->
